Week2_LibraryTask/.env

```diff
@@ -0,0 +1,4 @@
+INFURA_SEPOLIA_URL = YOUR_INFURA_SEPOLIA_URL
```

Comment:
I tried naming it ``.env.example``  but I couldn't access it through ``process.env`` and didn't look further into it. I need to research that a bit further. About the non-sensitive info, I just decided to make them such since the repo is public, but I do agree with you that I could have added the Etherscan API, and even the Infura one since both are from the free plans.

Reply:
Alright, It turns out I just had to specify the name of the .env file in ``dotenv.config``, the name is now updated. What would be good practice for the namings? Should I create a .env.example and push it to the remote repository and keep a .env.development with the actual data? If I do so, I would need to include that information in the README file.